CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes for each version of this project will be documented in this
 
 - `IgxGrid`, `IgxTreeGrid`, `IgxHierarchicalGrid`
     - **Behavioral Change** - Pinning columns is no longer automatically prevented when the pinning area would exceed the size of the grid.
+- `IgxGridExcelStyleFilteringComponent` and `IgxAdvancedFilteringDialogComponent` can now be hosted outside of the grid in order to provide the same experience as the built-in filtering UI.
```

Comment:
Maybe we can present this bullet point as a new feature in the **New Features** section.

Reply:
Done.